src/foam/nanos/approval/Approvable.js

```diff
@@ -85,6 +85,11 @@ foam.CLASS({
       class: 'Reference',
       of: 'foam.nanos.auth.User',
       name: 'createdByAgent'
+    },
+    {
+      class: 'Boolean',
+      name: 'isUsingNestedJournal',
```

Comment:
What does this mean?

Reply:
"Nested journal" is a term I made up on a whim to describe the FObjectArray of CapablePayload (because it's read/populated using a DAO interface). I believe @tharmaman used this property to control whether the approval is stored in this array instead of the invoice itself (it's a bit of a hacky solution but we haven't come up with anything better)